### DISCUSSION TITLE
Libre hosting trial expiring, pending approval

### DISCUSSION THREAD

**Discussion by dacmot** [WeblateOrg/weblate#12289] [Q&A] [answered]
Hello,

I created a project https://hosted.weblate.org/projects/whynot-game-for-minetest/ a few weeks ago. I filled in the request for libre hosting, and as far as I can tell, it meets all the criteria and I have nothing else to do. However, the trial is set to expire in less than a day. Am I missing something?

It's a small project without a lot of content. It's OK if it gets locked out, or even deleted because the trial is over. I understand if there's a big backlog of approvals to go through. I'm not in a rush. I would just like to avoid having to do the setup all over again.

Thanks.

**Comment by nijel** [comment]
Even if the trial expires, there will be an additional grace period until the project is actually removed. I'm sure that @orangesunny will get soon to reviewing your libre hosting request.

**Comment by orangesunny** [comment, marked as answer]
We work on that perpetually, and your hosting was just approved. Sorry for the 12 days of waiting. You should have been informed during, but your mail server didn’t want our messages. ⬇️ 

Well, don’t forget to mention you use Weblate in the README and the website of your project! This way, you’ll respect one of the guidelines for the free Libre plan you asked for (you should already), and it is also helping Weblate to grow and be there for more people in the libre community! And to get your project more translators! You can use the lovely content from the Share menu of your Weblate project for this presentation.

This information is in the e-mail we sent you, but your mail server rejected it:


```
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a permanent error. The following address(es) failed:

 [your mailserver](mailto:your address)
   host [mx.host.tld](http://your.website.tld/) [IP]
   SMTP error from remote mail server after RCPT TO:<[your address](mailto:your address)>:
   550 5.1.1 <[your address](mailto:your address)> recipient rejected
```

**Comment by dacmot** [comment]
Thank you for the quick reply and approval.

That is odd that the mail server returned a "permanent error." I have been using protonmail for several years and this is the first time I hear of email bouncing. Also, I did receive the notice that my trial period was about to expire (yesterday) so my email address is correct in my profile.

As for the README, a note with a link was added in the contribution section around the time I submitting the request for approval (https://github.com/minetest-whynot/whynot-game/commit/851cc2fa0172b4e46c7e88418102b177d088328c)
